src/main/java/jalau/at18/katas/game2048/team2/Board.java

```diff
@@ -0,0 +1,234 @@
+package jalau.at18.katas.game2048.team2;
+
+public class Board {
+    private int[][] board;
+    private int score;
+    private int sizeBoard;
```

Comment:
This attribute does not seem to be used, if so, please remove it

Reply:
removed